### PR TITLE
Seytkaliev Makshe Lab 2

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by DiazzzU** [comment]
Everything works

File: app_python/moscow_time/app.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from flask import Flask, render_template
+
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def index():
+    """Return inflated with current time index.html"""
+    date = datetime.now()
```
Comment (DiazzzU) [inline app_python/moscow_time/app.py:12]: Need to change to Moscow time, not local

**Comment by Kamila-Khamidullina** [comment]
Looks good to me

**Review by DiazzzU — COMMENTED** [review]
Everything looks good

**Pull request closed without merging**